docs/uptime/settings.asciidoc

```diff
@@ -25,3 +27,20 @@ to manually query Elasticsearch for data outside this pattern!
 See the <<uptime-security,Uptime Elasticsearch Security>>
 and {heartbeat-ref}/securing-heartbeat.html[Heartbeat security]
 docs for more information.
+
+=== Certificate thresholds
+
+The settings page also allows you to specify thresholds for all the TLS certificates monitored
+in your environment. There is a field for maximum age, and the minimum number of days before
+expiration before Uptime should consider a monitor to be in warning state.
+
+For example, you may want to make sure that none of your organization's TLS certificates have been
+valid for longer than one year. This is becoming a common security requirement. Modifying the `Age limit`
```

Comment:
```suggestion
For example, a common security requirement is to make sure that none of your organization's TLS certificates have been valid for longer than one year. Modifying the `Age limit`
```

Reply:
Updated in https://github.com/elastic/kibana/pull/66118/commits/4a63705487aec31499581bf529a31074dc718f4c.